apps/src/lib/kits/maker/WebSerialPortWrapper.js

```diff
@@ -24,7 +24,25 @@ export default class WebSerialPortWrapper extends EventEmitter {
     // TODO - not sure if this is used in Maker Toolkit yet
   }
 
-  // Opens the serial port and starts reading from port.
+  // Opens the serial port from Micro:Bit and starts reading from port.
+  openMBPort() {
```

Comment:
At this point in implementation, I create a separate function to open the serialport for the Micro:Bit since we are only opening the port here and not reading from or writing to the port. 
Later on when the entire WebSerial pathway is implemented, I may remove this function and include logic for the MB in the open() function below if appropriate.

Reply:
Plus +1 to Sanchit's comment! And I like having this temporarily broken out to preserve current behavior for CP and iterate on the MB pathway as needed!

Reply:
Thanks! Will do.